dockstore-webservice/src/main/java/io/dockstore/webservice/DockstoreWebserviceConfiguration.java

```diff
@@ -744,6 +744,12 @@ public void setWorkflowVersionLimit(int workflowVersionLimit) {
 
         private String composeSetupVersion;
 
+        private String cwlParsingLambdaVersion;
+        private String wdlParsingLambdaVersion;
+        private String nextflowParsingLambdaVersion;
+        private String galaxyParsingPluginVersion;
```

Comment:
Is the goal to make this one a lambda too rather than the external plugin it is now?

Reply:
hmm...never really thought about it. The language parsing lambdas are kind of disjointed from the plugin system (where others can more easily contribute)

Reply:
I think the challenge for cwl is that much of the re-usable code is in Python and for Nextflow in a full-blown jar that needs a new VM that has some substantial load implications. I think we can skip Galaxy since the plugin has been behaving pretty well and they didn't really suggest re-using anything.